docs/configuration.md

```diff
@@ -60,17 +60,18 @@ Property
-**envoy-control.envoy.snapshot.outgoing-permissions.all-services-dependencies.identifier**             | Special value (wildcard) that signifies that the service depends on all other services                                                                                                    | *
-**envoy-control.envoy.snapshot.outgoing-permissions.all-services-dependencies.not-included-by-prefix** | Services not included in dependencies for services with wildcard in outgoing.dependency field. Matched by service name prefix.                                                            | empty list
-**envoy-control.envoy.snapshot.outgoing-permissions.services-allowed-to-use-wildcard**                 | Services that are allowed to have wildcard in outgoing.dependency field                                                                                                                   | false
+Property                                                                                                | Description                                                                                                                                                                               | Default value
+------------------------------------------------------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ---------
+**envoy-control.envoy.snapshot.incoming-permissions.enabled**                                           | Enable incoming permissions                                                                                                                                                               | false
+**envoy-control.envoy.snapshot.incoming-permissions.client-identity-header**                            | Header that identifies the client that called the endpoint                                                                                                                                | x-service-name
+**envoy-control.envoy.snapshot.incoming-permissions.source-ip-authentication.enabled-for-services**     | Enable source ip based authentication for selected services                                                                                                                               | empty list
+**envoy-control.envoy.snapshot.incoming-permissions.tls-authentication.validation-context-config-name** | Name of validation context config, has to match static configuration in Envoy config                                                                                                      | validation_context
```

Comment:
I think we should also change name here (config-name -> secret-name). If I understand correctly, SDS provides "secrets" which are mapped to a specific name.